validate/error.go

```diff
@@ -0,0 +1,69 @@
+package validate
```

Comment:
I think image-tool could use the same OCIError struct. But OCIErrorCode can not be shared between runtime-tool and image-tool. I think we can keep it here.